doc/user/content/get-started/quickstart.md

```diff
@@ -12,297 +12,764 @@ aliases:
+    index only on the view(s) from which you will serve results may be
+    preferred.
+
+1. Create a view `flippers` to flag known flipper accounts. Flippers are those
+   users with more than 2 flipping activities.
 
     ```mzsql
-    SELECT * FROM fraud_activity LIMIT 100;
+    CREATE VIEW flippers AS
+    SELECT flipper_id
+    FROM flip_activities
+    GROUP BY flipper_id
+    HAVING count(*) >= 2;
     ```
 
-## Step 3. See results change!
+    To view a sample row, run the following query on `flippers`, run the
+    following [`SELECT`](/sql/select) command:
 
-**Operational work needs to surface and act on the most recent data.** The
-  moment your data changes, Materialize reacts. Let's verify that this _really_
-  happens by manually flagging some accounts as fraudulent, and observing
-  results change in real time!
+    ```mzsql
+    SELECT *
+    FROM flippers
+    LIMIT 10;
+    ```
+
+    As new data comes in for users matching the definition of a flipper,
+    Materialize keeps the `flippers` view up to date.
 
-1. Create a [**table**](/sql/create-table/) that allows you to manually flag
-fraudulent accounts.
+    *Optional exercise*. To see that this view can use the index created earlier, run an `EXPLAIN` on the query.
 
     ```mzsql
-    CREATE TABLE fraud_accounts (id bigint);
+    EXPLAIN
+    SELECT *
+    FROM flippers
+    LIMIT 10;
     ```
 
-1. In a new browser window, side-by-side with this one, navigate to the [Materialize console](https://console.materialize.com/),
-   and pop open another SQL Shell.
+    Depending upon your query patterns and usage, an existing index may be
+    sufficient. In other use cases, creating an index only on the view(s) from
+    which you will serve results may be preferred.
 
-1. To see results change over time, let's `SUBSCRIBE` to a query that returns
-the Top 5 auction winners, overall.
+1. Use the [`SUBSCRIBE`](/sql/subscribe/) command to see new flippers as new
+   data comes. [`SUBSCRIBE`](/sql/subscribe/) returns data from a source, table,
+   view, or materialized view as they occur, in this case, the view `flippers`.
+   The optional [`WITH (snapshot = false)` option](/sql/subscribe/#with-options)
+   indicates that the command displays only the new flippers that come in after
+   the start of the `SUBSCRIBE` operation, and not the flippers in the view at
+   the start of the operation.
 
    ```mzsql
    SUBSCRIBE TO (
-     SELECT buyer, count(*)
-     FROM winning_bids
-     WHERE buyer NOT IN (SELECT id FROM fraud_accounts)
-     GROUP BY buyer
-     ORDER BY 2 DESC LIMIT 5
-   );
+     SELECT *
+     FROM flippers
+   ) WITH (snapshot = false)
+   ;
    ```
 
-   You can keep an eye on the results, but these may not change much at the
-   moment. You'll fix that in the next step!
+   Because of the randomness of the data being generated, it may take some time
+   (~up to <red>1 minute</red>) for a new data to come in for users matching the
```

Comment:
👍 Sounds good.  Because I didn't know about the special tech that allows inputing data w/o opening a new browser window (thanks for the info!), I had used this example first because I didn't want people to open a new window.   But knowing that, I can replace this example with the one below.  This works out well because I did also want to have an optional exercise of rewriting one of the queries as a cte.

Reply:
Updated in patch.